assets/src/js/header.js

```diff
@@ -110,6 +110,74 @@ const closeElement = (event, buttonClass) => {
   closeButton.click();
 };
 
+/**
+ * Set the mobile tabs menu behavior on scroll
+ * Mobile tabs menu should hide when user scrolls down, and reappear when they scroll up
+ */
+const setMobileTabsMenuScroll = () => {
+  const menu = document.getElementById('nav-mobile');
+  if (!menu) {
+    return;
+  }
+
+  const distToClose = 50
+    , distToOpen = 50;
+  let lastScrollDir = null
+    , lastScrollTop = window.pageYOffset || document.documentElement.scrollTop
+    , lastScrollRef = lastScrollTop;
+
+  /**
+   * Get scroll direction, distance from the lastScrollTop, and distance from a reference point
+   *
+   * @return {Object} {scroll direction, top position, distance scrolled, distance scrolled from ref point}
+   */
+  const scrollData = () => {
+    let scrollTop = window.pageYOffset || document.documentElement.scrollTop;
```

Comment:
I see there are several places where you used `let` where you could have used `const` because the value is never changed, I think that would be better for readability